README.md

```diff
@@ -1 +1,63 @@
-# Hello-Microverse
\ No newline at end of file
+# :green_book: Table of Contents
+- [:book: About the Project](#about-project)
+  - [:hammer_and_wrench: Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [:computer: Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [:busts_in_silhouette: Authors](#authors)
+- [:telescope: Future Features](#future-features)
+- [:handshake: Contributing](#contributing)
+- [:star:️ Show your support](#support)
+- [:pray: Acknowledgements](#acknowledgements)
+# :book: [Hello Microverse project]
+**[Hello Microverse project]** is a simple project without any coding complex built with html and css that display a hello microverse message in your browser.
+## :hammer_and_wrench: Built With
+### Tech Stack
+**Client**
+React.js
+**Server**
+Express.js
+**Database**
+PostgreSQL
+### Key Features
+- **[When clicked it prints "Hello Microverse" in aquamarine color]**
+## :rocket: Live Demo
+- [No Live Demo Link]
+## :computer: Getting Started
+prints Hello Microverse in default browser.
+### Prerequisites
+Clone this repository to your desired folder and open with any browser.
```

Comment:
- [ ] Prerequisites are the things the user should know before being able to use the project.

## EXAMPLE

```
### Prerequisites
Basic knowledge of HTML, CSS, and JavaScript.
```